test/apps/iodemo/io_demo.cc

```diff
@@ -682,6 +719,12 @@ class P2pDemoCommon : public UcxContext {
 
     static void signal_terminate_handler(int signo)
     {
+        if ((unsigned)signo < ucs_static_array_size(signame)) {
+            LOG << "Run-time signal handling: " << signame[signo];
```

Comment:
IMO, printing just signal number is enough

Reply:
just ```SIGINT```?
may be add any word to simplify grep?

Reply:
I'd just print the number (2), prefer to not add all these constants to io_demo
Or, maybe Linux/glibc has some method to get signal name from number

Reply:
done